biz.aQute.bnd/src/aQute/bnd/main/RepoCommand.java

```diff
@@ -541,7 +541,7 @@ public void _diff(diffOptions options) throws UnsupportedEncodingException, IOEx
 					.put(map)
 					.flush();
 			else if (!options.diff())
-				bnd.printMultiMap(map);
+				MultiMap.format(map);
```

Comment:
This just makes a String and then discards it. Is that what you meant? I assume you want to print the String here?